tests/meson.build

```diff
@@ -161,7 +161,7 @@ if get_option('enable-movidius-ncsdk2')
 endif
 
 
-if get_option('enable-cppfilter')
+if get_option('enable-cppfilter') and gtest_dep.found()
```

Comment:
How about checking `gtest_dep.found()` inside each sub-directory? because such a dependency depends on each filter's test codes.. and for uniformity.

Reply:
Updated.